OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -1244,6 +1249,11 @@ await UpdateStatus(new()
 
     private async Task<Workshop> CheckDtoAndPrepareCreatedWorkshop(WorkshopCreateRequestDto dto)
     {
+        if(await featureManager.IsEnabledAsync("EnableWorkshopGroupTypeField"))
```

Comment:
one more comment i missed yesterday 😅 
it should check `EnableWorkshopGroupTypeField` is `false`, and only then do the hardcoding. If it is `true` - we need to re-use the data from dto without hardcoding.